app/ui/legacy/src/main/java/com/fsck/k9/fragment/MessageListFragment.java

```diff
@@ -1585,6 +1586,29 @@ private void copyOrMove(List<MessageReference> messages, long destinationFolderI
         }
     }
 
+    private void saveToDraftAndRemove(List<MessageReference> messages){
```

Comment:
`saveToDraftAndRemove` encodes the implementation detail in the name. I think a better name is `moveToDraftsFolder` because it is closer to the user-visible name of the action.